lib/WebLedgerClient.js

```diff
@@ -61,36 +61,41 @@ class WebLedgerClient {
 
     const ledgerAgent = await this.getAgent();
     const baseURL = ledgerAgent.service.ledgerQueryService;
-    const queryServiceApi =
-      create({baseURL, httpsAgent: this.httpsAgent, timeout: this.timeout});
 
     this.logger.log(`Retrieving Record [ledger: ${this.hostname}]:`, id);
-
-    const response = await queryServiceApi.post(
-      `?id=${encodeURIComponent(id)}`, {}, {
-        headers: constants.DEFAULT_HEADERS
+    let body = {};
+    try {
+      const response = await httpClient.post(baseURL, {
+        headers: DEFAULT_HEADERS,
```

Comment:
I don't think we need to pass in the `DEFAULT_HEADERS` here as it is already being baked into http-client https://github.com/digitalbazaar/http-client/blob/master/main.js#L10. So I think we can remove the `headers` param here and remove `DEFAULT_HEADERS` from everywhere it's being used, right?

Reply:
+1, this is just passing the `DEFAULT_HEADERS` already being used in http-client into HTTP client.  I note that this particular API not provide any options for providing any custom headers, but that is not something that needs to be addressed at this time.

Reply:
addressed here: https://github.com/digitalbazaar/web-ledger-client/pull/25/commits/b5f40a6329caa2b803b876a45f072ef76902120c